testsuite/features/support/lavanda.rb

```diff
@@ -151,15 +151,16 @@ def os_version
   #
   # Args:
   #   cmd: The command to run.
+  #   runs_in_container: Whether the command should be run in the the container or in the host. Defaults to true.
```

Comment:
```suggestion
  #   runs_in_container: Whether the command should be run in the container or on the host. Defaults to true.
```
Since `run()` is a wrapper for `run_local()` it makes sense to set this new parameter to `true` as default.